machine-learning/ctr-prediction/docs/more.md

```diff
@@ -1,8 +1,10 @@
 ## What the workflow did
 
-We prepared two workflows, `predict_logress.dig` and `predict_fm.dig`. In order to predict CTRs from a set of features, the former uses a well-known machine learning algorithm called **Logistic Regression**. An algorithm used in the latter workflow, namely **Factorization Machines**, is much more complex.
+We prepared three workflows, `predict_logress.dig`, `predict_fm.dig` and `predict_rf.dif`. In order to predict CTRs from a set of features, these workflows employ machine learning algorithm **Logistic Regression**, **Factorization Machines** and **Random Forest**, respectively.
 
-We recommend you to first try `predict_logress.dig` because the number of parameters we need to configure is smaller than `predict_fm.dig`, and exporting the models to external databases such as MySQL should be easy for further integration of learnt models and production ad servers.
+We recommend you to first try `predict_logress.dig` because the number of parameters we need to configure is smaller than `predict_fm.dig`, and exporting Logistic Regression model to external databases such as MySQL is easy for further integration of learnt models and production ad servers.
+
+**Caution:** Currently, Logistic Regression is much more stable and accurate compared to Random Forest in our platform due to an implementation-related issue of underlying OSS library, [Hivemall](https://github.com/apache/incubator-hivemall).
```

Comment:
Added a note about poor accuracy of RandomForest.

Reply:
Sampling scheme is different https://github.com/takuti-sandbox/tmp/commit/bad500bdd06fc497b3b0284c3778a574e7fcd74b#diff-1befd9516f7ec90288c644547380f288R32 

Also, feature pre-processing scheme is different to scikit's evaluation.
https://github.com/takuti-sandbox/tmp/commit/bad500bdd06fc497b3b0284c3778a574e7fcd74b#diff-1befd9516f7ec90288c644547380f288R29

So, we cannot conclude it's due to implementation issue. Also, we need to find/fix the implementation issue and no need to publish this article.